lib/Conversion/TTIRToTTMetal/TTIRToTTMetal.cpp

```diff
@@ -1343,8 +1350,15 @@ class TTIRToTTMetalDispatchRewriter : public OpRewritePattern<ttir::GenericOp> {
     //   return failure();
     // }
 
+    // FIXME - Currently metal requires UnpackToDestMode for all 32 CBs but will
+    // be relaxed soon to remove this requirement. Update logic here later.
```

Comment:
Worth adding this to the workarounds lib so we don't forget?

Reply:
Removed fix/hack, found we have ~equivalent define in our side already, used it instead:

```
constexpr uint32_t kNumCBs = 1 + ttkernel::getMaxEnumValForCBPort();
```

Will add assert here to make sure our copy matches metal copy.